services/director/requirements/_base.in

```diff
@@ -57,14 +57,14 @@ prometheus-client==0.8.0
 psycopg2-binary==2.8.6
 pydantic==1.7.2
 PyYAML==5.4 # CVE-2020-1747
-requests==2.22.0
+requests
```

Comment:
Could you pin back the library at version `requests==2.27.1` as seen below?

Reply:
i forgot that. thx